src/main/java/org/gradlex/javaecosystem/capabilities/rules/CapabilityDefinitions.java

```diff
@@ -392,6 +426,7 @@ public enum CapabilityDefinitions {
     ),
 
     JAVAX_WS_RS_API(HIGHEST_VERSION, JavaxWsRsApiRule.class,
+        "com.sun.jersey:jersey-core",
```

Comment:
I'm trusting the [pom](https://github.com/scijava/pom-scijava/blob/master/pom.xml#L4493-L4498)

Reply:
I don't know why the comment is in that POM, but here this is wrong. There are no overlapping classed. This would break a lot of builds because classes will be missing.

In this plugin, it is important to only add capabilities when there really is a conflict because two Jars contain the same classes.

You can test this, by removing a rule again and then run the `SamplesTest` (sample-all_build.sample).

If it gives you an error like:

```
Collision detected! Entry org/apache/commons/beanutils/locale/converters/BigDecimalLocaleConverter.class present in following JARs:
```
...the rule is valid.

(We should have some better mechanism that tests this automatically in the future - #30)